### DISCUSSION TITLE
Multiple submit button in form

### DISCUSSION THREAD

**Discussion by bastiengrignon** [mantinedev/mantine#6703] [Q&A] [answered]
Is it possible to add multiple submit button that are submitting the form to different endpoints for example ? 
Or is there a way to achieve this behavior, I haven't found any submit function for the form only one that validate it.

**Comment by rtivital** [comment, marked as answer]
You can get form values at any time with `form.getValues()`. To validate fields, use `form.validate()`. Using these two function you can create any custom logic when button is clicked. For example:

```tsx

<>
<Button onClick={() => callback1(form.getValues())}>First</Button>
<Button onClick={() => callback2(form.getValues())}>Second</Button>
</>
```
